extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -58,7 +58,7 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 	 * @param location the parent directory
 	 * @returns Uri of the newly created project file
 	 */
-	async createProject(name: string, location: vscode.Uri): Promise<vscode.Uri> {
+	async createProject(name: string, location: vscode.Uri, _: string): Promise<vscode.Uri> {
```

Comment:
IIUC, you are trying to avoid the unused variable error, right？

Reply:
I think if you are implementing an interface, it won't complain about unused parameters.

Reply:
Yup.  In a different PR  (https://github.com/microsoft/azuredatastudio/pull/13558), I update the extension to actually use this variable.  Since we'll be releasing the updated extension out of main, that's not part of this port request.

Reply:
Even if the interface has it as a required param?  Huh, that seems unintuitive...